stable/postgresql/templates/statefulset.yaml

```diff
@@ -185,7 +185,11 @@ spec:
         volumeMounts:
         {{- if or (.Files.Glob "files/docker-entrypoint-initdb.d/*.{sh,sql,sql.gz}") .Values.initdbScriptsConfigMap .Values.initdbScripts }}
         - name: custom-init-scripts
-          mountPath: /docker-entrypoint-initdb.d
+          mountPath: /docker-entrypoint-initdb.d/configmap
```

Comment:
can you mount some scripts at `configmap` folder, others at `secret` folder. and postgresql will take all of them?

Reply:
Yes, as you can see here https://github.com/bitnami/bitnami-docker-postgresql/blob/master/9.6/debian-9/rootfs/run.sh

Reply:
ok!